src/content/tools/pub/cmd/pub-upgrade.md

```diff
@@ -145,5 +154,10 @@ Updates the lower bounds of dependencies in `pubspec.yaml` to match the
 resolved versions, and returns a list of the changed constraints. 
 Can be applied to [specific dependencies](#upgrading-specific-dependencies).  
 
+### `--unlock-transitive`
+
+When used with a list of packages to unlock, first the transitive closure of
+those packages' dependencies (in the current resolution) is computed, and all
+those packages are unlocked.
```

Comment:
```suggestion
those packages' dependencies (in the current resolution) is computed,
and then all those packages are unlocked.
```
(since you said "first...", waiting for a "then...")